Code/GraphMol/MolStandardize/Validate.h

```diff
@@ -80,12 +80,17 @@ class RDKIT_MOLSTANDARDIZE_EXPORT CompositeValidation
 */
 class RDKIT_MOLSTANDARDIZE_EXPORT RDKitValidation : public ValidationMethod {
  public:
+  RDKitValidation(bool allowEmptyMolecules = false)
+      : allowEmptyMolecules(allowEmptyMolecules){};
+
   std::vector<ValidationErrorInfo> validate(
       const ROMol &mol, bool reportAllFailures) const override;
 
   std::shared_ptr<ValidationMethod> copy() const override {
     return std::make_shared<RDKitValidation>(*this);
   }
+
+  bool allowEmptyMolecules;
```

Comment:
Ok.. I thought it could be unnecessary because the constructor has a default argument.

If I initialize it in the class definition, should I do the same for the other `ValidationMethod` subclasses that have data members?

Reply:
> Ok.. I thought it could be unnecessary because the constructor has a default argument.
> 
> If I initialize it in the class definition, should I do the same for the other `ValidationMethod` subclasses that have data members?

yeah, it's fine. I needed to go check at cppreference.com to check the rules for default constructors.